app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -207,7 +207,7 @@ def parse_vapp_template(vapp_template)
       :ems_ref     => uid,
       :name        => vapp_template.name,
       :description => vapp_template.description,
-      :orderable   => false,
+      :orderable   => true,
```

Comment:
@gberginc vapp templates are discovered independent of vapps (stacks), right? How do you link a stack to its template?